gadget/params.c

```diff
@@ -231,23 +235,23 @@ create_gadget_parameter_set()
     param_declare_int(ps, "SnapshotWithFOF", REQUIRED, 0, "Enable Friends-of-Friends halo finder.");
     param_declare_int(ps, "FOFSaveParticles", OPTIONAL, 1, "Save particles in the FOF catalog.");
     param_declare_double(ps, "FOFHaloLinkingLength", OPTIONAL, 0.2, "Linking length for Friends of Friends halos.");
-    param_declare_int(ps, "FOFHaloMinLength", OPTIONAL, 32, "");
+    param_declare_int(ps, "FOFHaloMinLength", OPTIONAL, 32, "Minimum number of particles per FOF Halo.");
     param_declare_double(ps, "MinFoFMassForNewSeed", OPTIONAL, 5e2, "Minimal Mass for seeding tracer particles ");
     param_declare_double(ps, "TimeBetweenSeedingSearch", OPTIONAL, 1e5, "Time Between Seeding Attempts: default to a a large value, meaning never.");
 
     /*Black holes*/
     param_declare_int(ps, "BlackHoleOn", REQUIRED, 1, "Enable Blackhole ");
-    param_declare_double(ps, "BlackHoleAccretionFactor", OPTIONAL, 100, "");
-    param_declare_double(ps, "BlackHoleEddingtonFactor", OPTIONAL, 3, "");
-    param_declare_double(ps, "SeedBlackHoleMass", OPTIONAL, 5e-5, "");
+    param_declare_double(ps, "BlackHoleAccretionFactor", OPTIONAL, 100, "BH accretion rate as a fraction of the Bondi accretion rate.");
```

Comment:
BH accretion boosting factor relative to the rate from the Bondi accretion model.